packages/harmony/src/components/comments/CommentBody/CommentBody.tsx

```diff
@@ -0,0 +1,52 @@
+import { useRef, useState } from 'react'
+
+import { useEffectOnce } from 'react-use'
+
+import { Flex } from 'components/layout'
+import { Text } from 'components/text'
+import { TextLink } from 'components/text-link'
+
+import { CommentBodyProps } from './types'
+
+const messages = {
+  seeMore: 'See More',
+  seeLess: 'See Less'
+}
+
+export const CommentBody = ({ children }: CommentBodyProps) => {
+  const textRef = useRef<HTMLElement>()
+  const [isOverflown, setIsOverflown] = useState(false)
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  useEffectOnce(() => {
+    setIsOverflown(
+      (textRef.current &&
+        textRef.current.offsetHeight < textRef.current.scrollHeight) ||
+        false
+    )
+  })
+
+  return (
+    <Flex direction='column' alignItems='flex-start' gap='xs'>
+      <Text
+        // @ts-ignore
```

Comment:
Remove TS ignore? If it's the `null` issue, probably just need to add null into `useRef(null)`

Reply:
nah this one is a HtmlElement vs ElementType issue bc of the ref setup of the Text component

Reply:
hmm, fun. Well if its gross to fix then just a quick comment is good too!